src/main/java/com/virtuslab/using_directives/custom/Parser.java

```diff
@@ -266,11 +255,14 @@ UsingPrimitive primitive(int keyEnd) {
       } else {
         error(String.format("-%s is not a valid numeric literal. %s", in.td.name, solution));
       }
-    } else if (!in.td.isAfterLineEnd() && in.td.token == Tokens.IDENTIFIER) {
-      error(
-          String.format(
-              "Expected primitive value: string, numeric or boolean but found %s. %s",
-              in.td.toTokenInfoString(), solution));
+    } else if (in.td.token == Tokens.IDENTIFIER) {
+      if (in.td.name.startsWith("-") && in.td.name.chars().allMatch(Character::isDigit)) {
```

Comment:
i am not sure I understand. `-42` begins with `-`, but `-` Is not a digit so this check will not pass?

Reply:
Yeah, you are right, it doesn't work. I wonder though should we care about numbers. I changed it for now, but it's a bit hacky to make it work. Should we maybe drop numeric values? It can be parsed later on if needed.